src/components/MovieCard/MovieCard.tsx

```diff
@@ -1,8 +1,9 @@
 import React from 'react';
 import './MovieCard.scss';
+// import { getAllMovies, targetMovie } from '../../api/movies';
 
 type Props = {
-  movie: Movie;
+  movie: Movie | null;
```

Comment:
you already checked if the movie exists in FindMovie component
```suggestion
  movie: Movie;
```